assets/src/navLinkData.ts

```diff
@@ -10,11 +11,23 @@ export interface LinkData {
   path: string
   navIcon: React.JSXElementConstructor<HTMLElementProps>
   onClick?: () => void
+  hideOnMobile?: boolean
 }
 
 export const getNavLinkData: () => LinkData[] = () => {
   const mapMode = mapModeForUser()
 
+  const maybeDetours = inTestGroup(TestGroups.DetourPanel)
```

Comment:
This is a silly name, and I don't _hate_ it, but maybe it would be better to call it like... `optionalDetoursLink` or something like that?

Reply:
I actually think it's fine, to be honest.